src/Gold.Redis/Gold.Redis.LowLevelClient/Communication/SocketsConnectionsContainer.cs

```diff
@@ -1,99 +1,108 @@
-﻿using Gold.Redis.Common.Interfaces.Communication;
```

Comment:
if you could, please do this with smaller commits so that git will recognize your actual diff